python/shark_turbine/dynamo/importer.py

```diff
@@ -65,6 +65,11 @@
     "FxImporter",
 ]
 
+# An external callback that, given a Python value and a GraphNodeImporter, may choose
+# to materialize IR to load the value as a vtensor. If it returns None, then default
+# literal resolution proceeds.
+LiteralResolverCallback = Callable[[Any, "GraphNodeImporter"], Optional[Value]]
+
 REQUIRED_DIALCTS = [
```

Comment:
is this not dialects? Sorry this is unrelated to your patch, just looked weird to me

Reply:
Indeed. Typo. Patches welcome :)